RefRed/config/configobj.py

```diff
@@ -2069,12 +2057,12 @@ def write(self, outfile=None, section=None):
         if not output.endswith(newline):
             output += newline
         if outfile is not None:
-            outfile.write(u_str(output))
+            outfile.write(str(output))
         else:
             h = open(self.filename, 'wb')
             if type(output) is not bytes:
                 output=output.encode() # encoding the data to bytes
-            h.write(u_str(output))
+            h.write(str(output))
```

Comment:
When I try to start RefRed I'm getting

```
Error in atexit._run_exitfuncs:
Traceback (most recent call last):
  File "/SNS/users/rwp/src/RefRed/RefRed/config/baseconfig.py", line 223, in store
    item.write()
  File "/SNS/users/rwp/src/RefRed/RefRed/config/configobj.py", line 2065, in write
    h.write(str(output))
TypeError: a bytes-like object is required, not 'str'
```

Can we remove `configobj` since it's just copied from https://github.com/DiffSK/configobj and add it as a dependency? This way we don't need to maintain this code.

Reply:
I'll check this `configobj.py` file is an exact duplicate of the one you pointed out

Reply:
It won't be an exact duplicate, it's a newer version that supports python 3. You should be able to `pip install configobj`. It's even a system package on Ubuntu `python3-configobj`. So I don't see the need to keep and maintain an old version.

Reply:
yes, I added it as a dependency.
I looked at python standard parser (https://docs.python.org/3/library/configparser.html) but after reading a little, it looks like people prefer configobj (more intuitive and better validation) so I guess we can keep it as a dependency